### DISCUSSION TITLE
How can I Delete a Collection?

### DISCUSSION THREAD

**Discussion by adreinhard** [knowledgefutures/pubpub#2154] [Support] [answered]
Hi--PubPub n00b here. I just created my first Collection, messed up the title, and cannot find a way to delete the Collection after the save. As a workaround I tried renaming the Collection by editing its metadata, but the change didn't save. Basically, I want to either: 1) create a Collection for my journal, then create sub-collections for each issue of that journal, or 2) create separate Collections for each journal issue, linking those to a Community. But I messed up during Collection-creation and want to start over. So, how do I delete a Collection? Many thanks.

**Comment by gabestein** [comment, marked as answer]
Hello and welcome to PubPub! Thanks for taking it for a spin. To answer your larger question: for journals, PubPub is more or less designed to host one journal per Community. There are some ways around it if you want to host multiple journals within a single community, but typically that's what we recommend. You can learn more about what we recommend on our Getting Started guide for journals on our help site: https://help.pubpub.org/pub/creating-a-journal-on-pubpub/release/1

On your specific questions: the saving issue is probably related to this bug, which has a workaround: #2061. We're also currently working on fixing this and streamlining settings screens across the app!

To delete the collection, if that's easier for you, visit its settings page in the dashboard and click the "Delete" button under details.

<img width="1000" alt="Screen Shot 2022-07-28 at 16 30 06" src="https://user-images.githubusercontent.com/639110/181631579-94ab0d67-28d9-481a-99e2-e465ea0703af.png">

Let us know if we can help with anything else!

**Reply by adreinhard** [reply]
Thank you--that worked. I appreciate the time you took to answer my question. It makes sense that each journal would be its own community, but in our case my organization publishes three. It would be cool to have a community for my organization and then separate pages for each journal's info, then collections for each of the journals' issues. To start, however, I'll try creating a Community for one journal and collections for each issue of that journal.
